guppylang-internals/src/guppylang_internals/std/_internal/compiler/prelude.py

```diff
@@ -90,7 +98,7 @@ def build_panic(
     return builder.add_op(op, err, *args)
 
 
-def build_error(builder: DfBase[P], signal: int, msg: str) -> Wire:
+def build_static_error(builder: DfBase[P], signal: int, msg: str) -> Wire:
```

Comment:
I think this rename technically makes this a breaking change for `guppy_internals`. Not sure why the semver check didn't catch it?